components/Themed.tsx

```diff
@@ -125,14 +136,19 @@ export function LinkButton(props: LinkButtonProps) {
 
   return (
     <DefaultTouchableOpacity onPress={onPress}>
-      <DefaultText style={[{color: textColor, fontSize: 24}, style]}>
+      <DefaultText style={[{color: textColor, fontSize: 20}, style]}>
         {title}
       </DefaultText>
     </DefaultTouchableOpacity>
 
   );
 }
 
+LinkButton.defaultProps = {
+  lightColor: '#2684FF',
+  darkColor: '#fff',
```

Comment:
Should this go in/be sourced from `constants/Colors.ts`? I thiink the point of the lightColor/darkColor props is as an override to the default themed components

Reply:
Oh yeah, good catch - I didn't quite get that light/dark color props thing at first.  Moved it over.